tests/blackbox/stratis_cli_cert.py

```diff
@@ -140,7 +141,7 @@ def test_pool_list_not_empty(self):
         """
         Test listing an existent pool.
         """
-        make_test_pool()
+        make_test_pool(DISKS[0])
```

Comment:
Make sure to always use a slice when selecting from the DISKS array, as you do in stratisd_cert.py. A slice of a list of strings is itself a list of strings, but an element of a list of strings is just a string.